src/main/scala/com/fulcrumgenomics/vcf/HapCutToVcf.scala

```diff
@@ -274,10 +326,28 @@ private case class HapCutBlockInfo (private val phaseSetOption: Option[Int] = No
   def phaseSet: Int = phaseSetOption.get
 }
 
+private object GenotypeInfo {
+  def apply(info: String, hapCut2: Boolean): GenotypeInfo = {
+    if (!hapCut2) {
+      HapCutGenotypeInfo(info=info)
```

Comment:
Where you have bifurcations like this it might be clearer to have `HapCut1GenotypeInfo` so it's clear the class is HapCut1 specific, and not general to both versions?